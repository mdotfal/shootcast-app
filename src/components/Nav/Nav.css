@import url("https://use.typekit.net/yar1vam.css");
@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@400;500&display=swap');
@supports (-webkit-text-stroke: 1px black) {
  .stroke {
    -webkit-text-stroke: 1px black;
    -webkit-text-fill-color: rgb(255, 181, 43);
  }
}

/* Nav Styles */

.nav {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 10px;
  overflow: auto;
  border-radius: 25px;
  background-color: #5e351b;
  max-width: 100%;
  color: white;
  margin: 5px;
}

.nav-logo {
  font-family: sarina, sans-serif;
  font-weight: 400;
  font-style: normal;
  font-size: 30px;
  color: rgb(255, 181, 43);
  text-shadow: 2px 2px #000000;
}

.nav-links {
  font-family: 'Noto Sans JP', sans-serif;
  display: flex;
  flex-direction: row;
  margin-left: 15px;
}

.nav a {
  padding: 0;
  color: #eee;
  text-decoration: none;
  margin: 12px 10px;
  font-size: 14px;
}

@media all and ( min-width: 600px ) {
  .nav {
    width: 100vw;
    max-width: 90%;
    margin: 25px 25px;
    justify-content: space-around;
  }

  .nav-logo {
    font-size: 50px;
  }

  .nav a {
    font-size: 18px;
  }
}