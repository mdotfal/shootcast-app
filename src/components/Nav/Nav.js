import React from 'react';
import { Link } from 'react-router-dom';
import './Nav.css';

function Nav() {

  const signOut = () => {
    window.location.reload( false );
  } 

  return (
    <div className="nav">
      <Link to='/home'>
        Home  
      </Link>
        <p onClick={ signOut }>Sign Out</p>
    </div>
  );
};

export default Nav;