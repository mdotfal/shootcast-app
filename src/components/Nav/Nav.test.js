import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Nav from './Nav';


describe( `Nav component`, () => {
  it(Â `renders without errors`, () => {
    const wrapper = shallow( <Nav /> )
    expect( toJson( wrapper )).toMatchSnapshot()
  })
})