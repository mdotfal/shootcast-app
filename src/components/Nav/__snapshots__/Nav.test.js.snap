// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component renders without errors 1`] = `
<div
  className="nav"
>
  <div
    className="nav-logo"
  >
    ShootCast
  </div>
  <div
    className="nav-links"
  >
    <Link
      to="/home"
    >
      Home
    </Link>
    <Link
      to="/"
    >
      Sign Out
    </Link>
  </div>
</div>
`;
